File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P11_ConvertSpeedUnits/P11_ConvertSpeedUnits.cs
using System;

namespace P11_ConvertSpeedUnits
{
    class P11_ConvertSpeedUnits
    {
        static void Main(string[] args)
        {
            float meters = float.Parse(Console.ReadLine());
            byte hours = byte.Parse(Console.ReadLine());
            byte minutes = byte.Parse(Console.ReadLine());
            byte seconds = byte.Parse(Console.ReadLine());
            float sumSeconds = (hours * 3600) + (minutes * 60) + seconds;
            float sumMinutes = (sumSeconds / 3600) % 60;

            float km = meters / 1000;
            float miles = meters / 1609;

            Console.WriteLine((meters / sumSeconds) + Environment.NewLine + (km / sumMinutes) + Environment.NewLine + 
                (miles / sumMinutes));
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P11_ConvertSpeedUnits.java
import java.math.RoundingMode;
import java.text.DecimalFormat;
import java.util.Scanner;
public class P11_ConvertSpeedUnits {
    public static void main(String[] args) {

    //Doesn't work in judge, the tests are made for c#...

        Scanner console = new Scanner(System.in);
        float meters = Float.parseFloat(console.nextLine());
        byte hours = Byte.parseByte(console.nextLine());
        byte minutes = Byte.parseByte(console.nextLine());
        byte seconds = Byte.parseByte(console.nextLine());
        float sumSeconds = (hours * 3600) + (minutes * 60) + seconds;
        float sumMinutes = (sumSeconds / 3600) % 60;

        float km = meters / 1000;
        float miles = meters / 1609;
        //DecimalFormat df = new DecimalFormat("#.######");
        //df.setRoundingMode(RoundingMode.HALF_UP);
        System.out.format(/*df.format*/((meters / sumSeconds)) + System.lineSeparator() +
                            /*df.format*/((km / sumMinutes)) + System.lineSeparator() +
                            /*df.format*/((miles / sumMinutes)));
    }
}